config.json

```diff
@@ -900,7 +900,12 @@
       "core": false,
       "difficulty": 8,
       "slug": "book-store",
-      "topics": null,
+      "topics": [
+        "integers",
+        "lists",
+        "algorithms",
+        "mathematics"
```

Comment:
Great suggestions! I would add `floating_point_numbers` as well since the price returned is a double :)

Reply:
good point, I'm adding it 👍